src/hooks/useFirestoreConnect.ts

```diff
@@ -40,9 +55,8 @@ export interface SparkleRFQConfig extends ReduxFirestoreQuerySetting {
  * @see ValidFirestoreKeys
  * @see ReduxFirestoreQuerySetting
  */
-export const useSparkleFirestoreConnect = (
-  config?: SparkleRFQConfig[] | SparkleRFQConfig
-) => useFirestoreConnect(config);
+export const useFirestoreConnect = (config: SparkleRFQConfig) =>
+  _useFirestoreConnect(config);
```

Comment:
This is a better way to implement this, and to not include `undefined` in the definition of `SparkleRFQConfig`
```suggestion
export const useFirestoreConnect = (config?: SparkleRFQConfig) =>
  _useFirestoreConnect(config);
```

Reply:
This was useful when we still had separated queries. And a query could be undefined. 

`const query: SparkleRFQConfig = userId ? { querydata } : undefined;`

Reply:
https://github.com/sparkletown/sparkle/pull/801#discussion_r557267695

We're still able to use those `undefined` queries because of the `?` 'optional' operator.

If we still had separated queries, my preference would be fore them to be explicit rather than 'hiding' the fact that they contain `undefined`:

```
const query: SparkleRFQConfig | undefined = userId ? { querydata } : undefined;
```